english/boston/boston.md

```diff
@@ -2,4 +2,7 @@
 
 "This city is the life! Though now that I'm here, I think I'll take a vacation..."
 
+...or on the second thought, I'll put on a coat, stop being a sissy, and get to work!
+
 [Go on vacation](../licked-cat/vacation.md)
+[Get to work](../gtd/gtd.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.